source/_components/sensor.zamg.markdown

```diff
@@ -25,24 +25,55 @@ sensor:
   - platform: zamg
 ```
 
-Configuration variables:
-
-- **station_id** (*Optional*): The ID number for a supported ZAMG station.
-- **name** (*Optional*): Additional name for the sensors. Defaults to platform name.
-- **latitude** (*Optional*): Latitude coordinate to monitor weather of (required if **longitude** is specified). Defaults to coordinates defined in your `configuration.yaml` file.
-- **longitude** (*Optional*): Longitude coordinate to monitor weather of (required if **latitude** is specified). Defaults to coordinates defined in your `configuration.yaml` file.
-- **monitored_conditions** array (*Optional*): Conditions to display in the frontend.
-  - **pressure**: Pressure at station level
-  - **pressure_sealevel**: Pressure at sea Level
-  - **humidity**: Humidity
-  - **wind_speed**: Wind speed
-  - **wind_bearing**: Wind bearing
-  - **wind_max_speed**: Top wind speed
-  - **wind_max_bearing**: Top wind bearing
-  - **sun_last_hour**: Sun last hour percentage
-  - **temperature**: Temperature
-  - **precipitation**: Precipitation
-  - **dewpoint**: Dew point
+{% configuration %}
+station_id:
+  required: false
+  description: The ID number for a supported ZAMG station.
+  type: string
```

Comment:
Actually this should be an integer but mad enough I see in the python code that there is a string for it 🤔

Reply:
The website returns a CSV, so everything is a string.

See: https://github.com/home-assistant/home-assistant/blob/07972c84a15740a8de47883749f7eb9301bcdc89/homeassistant/components/sensor/zamg.py#L178

Reply:
Do not say that it should be an integer but it could be 😉

Reply:
I am with you on that.